pkgs/games/keen4/builder.sh

```diff
@@ -1,10 +1,13 @@
-source $stdenv/setup
+#!/usr/bin/env bash
 
-mkdir -p $out/share/keen4
-unzip -j $dist -d $out/share/keen4
+# shellcheck source=/dev/null
+source "${stdenv:?}"/setup
 
-mkdir -p $out/bin
-cat > $out/bin/keen4 <<EOF
+mkdir -p "${out:?}"/share/keen4
+unzip -j "${dist:?}" -d "${out:?}"/share/keen4
```

Comment:
ShellCheck suggested `"$out"/share/keen4`, so I've gone with that even though I personally prefer "${out}/share/keen4". Do you have a preference either way?

Reply:
I prefer quoted whole too.

Reply:
Quoted whole strings instead.